squid-gen-ink/support/util.ts

```diff
@@ -1,3 +1,18 @@
+import * as ss58 from '@subsquid/ss58'
+import {toHex} from '@subsquid/util-internal-hex'
+
+export function fromss58(addr: string) {
+    let decodedAddr = ss58.decode(addr)
+    return {
+        prefix: decodedAddr.prefix,
+        hexAddress: toHex(decodedAddr.bytes)
+    }
+}
+
+export function toss58(prefix: number, hexAddr: string) {
+    return ss58.encode({prefix, bytes: Uint8Array.from(Buffer.from(hexAddr.substr(2), 'hex'))})
```

Comment:
if we anyway depend on `util-internal-hex` then it's better to use [import {toHex} from '@subsquid/util-internal-hex'](https://github.com/subsquid/squid-sdk/blob/master/util/util-internal-hex/src/hex.ts#L18) because it provides additional assertion

Reply:
and I guess if we use `toHex` there is no need to wrap `encode` into another function